--- origsrc/okular-4.10.5/active/components/CMakeLists.txt	2013-06-28 12:37:20.176808700 -0500
+++ src/okular-4.10.5/active/components/CMakeLists.txt	2013-08-28 09:55:17.346358300 -0500
@@ -22,7 +22,7 @@ kde4_add_kcfg_files(okular_SRCS ${CMAKE_
 
 qt4_automoc(${okular_SRCS})
 
-kde4_add_library(okularplugin SHARED ${okular_SRCS})
+kde4_add_library(okularplugin MODULE ${okular_SRCS})
 target_link_libraries(okularplugin
         ${QT_QTCORE_LIBRARY}
         ${QT_QTGUI_LIBRARY}
